extensions/ql-vscode/src/local-databases-ui.ts

```diff
@@ -390,6 +390,7 @@ export class DatabaseUI extends DisposableObject {
           );
         }
         await this.databaseManager.setCurrentDatabaseItem(databaseItem);
+        await this.handleTourDependencies();
```

Comment:
Minor: Arguably this should be called at a higher level, not as part of `handleSetDefaultTourDatabase`.